site/app/templates/calendar/Calendar.twig

```diff
@@ -6,6 +6,8 @@
     const curr_month = {{ curr_month }};
     const curr_day = {{ curr_day }};
     const instructor_courses = {{ instructor_courses|json_encode|raw }};
+    const global_items_by_date = {{global_items_by_date|json_encode|raw }};
+    const is_superuser = {{ is_superuser ? "true" : "false" }};
```

Comment:
This is inside of a script, so having the const is_superuser doesn't affect the twig later in this file. In a later PR,  would it be better to move this JS script into the calendar.js file?

Reply:
That `const is_superuser` is referenced here: https://github.com/Submitty/Submitty/blob/7f397a530ff73531df6ecae0a68a45808f2e4488/site/public/js/calendar.js#L246

So it is nessesary to have a `is_superuser` variable, at least the way this code is currently set up. We could theoretically move the script to a js file but then we would need a different way to allow the php backend to let `calendar.js` know if the currently logged in user is a superuser or not.